nasa-gibs/worldview
# WV-2824 Removing HLS True Color Presets And DDV Layers

## Description

Removing the HLS true color preset for both Sentinel and Landsat. Also removing the HLS True Color Landsat/Sentinel layers from the layer picker. 

## How To Test

1. `git checkout wv-2824`
2. `npm ci`
3.  `npm run build`
4. `npm run watch`
5.  Verify that the HLS True Color Landsat/Sentinel layers are not available in the layer picker
6. Add the `HLS Customizable Landsat` and `HLS Customizable Sentinel` layers 
7. Open the layer options for each layer and verify that `True Color` no longer a selectable preset. 

@nasa-gibs/worldview
